src/plugins/drawing/objects/drawing-object.tsx

```diff
@@ -226,7 +226,8 @@ export interface IDrawingLayer {
   selectTile: (append: boolean) => void;
   getWorkspacePoint: (e:MouseEvent|React.MouseEvent) => Point|null;
   setCurrentDrawingObject: (object: DrawingObjectType|null) => void;
-  addNewDrawingObject: (object: DrawingObjectSnapshotForAdd) => DrawingObjectType;
+  addNewDrawingObject:
+    (object: DrawingObjectSnapshotForAdd, addAtBack?: boolean, select?: boolean) => DrawingObjectType;
```

Comment:
I wonder if it would be better at this point to have a `addOptions` parameter that contains `addAtBack` and `select` rather than adding more sequential parameters.